DuckDuckGo/NetworkProtectionStatusViewModel.swift

```diff
@@ -138,7 +140,7 @@ final class NetworkProtectionStatusViewModel: ObservableObject {
         didSet {
             if #available(iOS 17.0, *) {
                 if isNetPEnabled {
-                    VPNGeoswitchingTip.donateVPNConnectedEvent()
+                    VPNGeoswitchingTip.vpnEnabledOnce = true
```

Comment:
Just a change to keep this as a flag, rather than an ever growing counter.  We just need to know this happened once.